2-projects/1-todo-list/script.js

```diff
@@ -2,7 +2,23 @@ function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
 }
+function toggleIsComplete(todo) {
+   todo.completed = !todo.completed;
+ }
 
+ function greenrawTodo(li, isComplete) {
+   if (isComplete) {
+     li.style.textDecoration = "line-through";
+     li.style.color = "green";
+   } else {
+     li.style.textDecoration = null;
+     li.style.color = "white";
+   }
+ }
+
+ function deleteTodo(li) {
+   document.querySelector("ol").removeChild(li);
```

Comment:
this will just select the first `ol` in the html - how could you make this more specific to get the `todo-list` element you want?